api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -104,6 +104,12 @@ type HeaderSettings struct {
 	// +optional
 	EnableEnvoyHeaders *bool `json:"enableEnvoyHeaders,omitempty"`
 
+	// DisableRateLimitHeaders configures Envoy Proxy to omit the "X-RateLimit" headers.
+	// If DisableRateLimitHeaders is false, "X-RateLimit" headers will be emitted.
+	// If not set, DisableRateLimitHeaders is False.
+	// +optional
+	DisableRateLimitHeaders bool `json:"disableRateLimitHeaders,omitempty"`
```

Comment:
lets make this `*bool` /  ptr to follow https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required